backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc

```diff
@@ -18,4 +18,6 @@ include::modules/dr-scenario-cluster-state-issues.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-* See xref:../../../networking/accessing-hosts.adoc#accessing-hosts[Accessing the hosts] for how to create a bastion host to access {product-title} instances and the control plane nodes with SSH.
+* xref:../../../networking/accessing-hosts.adoc#accessing-hosts[Accessing the hosts] for how to create a bastion host to access {product-title} instances and the control plane nodes with SSH.
```

Comment:
You could reduce this cross-reference as follows, so that there is parallelism between the two items in this "Additional resources" section:

* xref:../../../networking/accessing-hosts.adoc#accessing-hosts[Accessing the hosts]